src/pages/dev/send-tokens/interchain-tokens/create-token.mdx

```diff
@@ -56,7 +56,19 @@ To create a custom Interchain Token:
 
 - Build your [ERC-20](https://docs.alchemy.com/docs/how-to-create-an-erc-20-token-4-steps) token and deploy it on multiple chains
 - Deploy a [Mint/Burn](../../reference/glossary/#mintburn) - [Token Manager](../../reference/glossary/#token-manager) for existing tokens on all chains using [`deployTokenManager`](https://github.com/axelarnetwork/interchain-token-service/blob/9edc4318ac1c17231e65886eea72c0f55469d7e5/contracts/InterchainTokenService.sol#L276) on Interchain Token Service. The `deployTokenManager` method requires parameters like `salt`, `destinationChain`, `tokenManagerType`, `params`, and `gasValue`
-- Make sure the Interchain Token Service can call the mint/burn on the token and repeat the step above for other chains where the token exists using the same deployer address and salt
+- Ensure the Interchain Token Service can invoke the `mint/burn` functions on the token by calling [`transferMintership`](https://github.com/axelarnetwork/interchain-token-service/blob/9223108211b977d9138fdd67d5b4a641fc35c18c/contracts/utils/Minter.sol#L31) on your token. This is to transfer the minter role to ITS on all chains, enabling it to mint/burn. Repeat this step for other chains where the token exists, using the same deployer address and salt 
```

Comment:
What do we think about linking to master vs a sha?

Reply:
I recommend referencing the master/main for everything. However, copying a permalink that points to `transferMintership` on the contract creates a SHA and links to it.